Upgrade CLI to v7

sembrestels opened the pull request:
We are testing budler but it is also important to have a branch of the app to latest aragon CLI just in case we need a backup.

I updated it to aragon-cli v7 but I have some issues with the `mock-data` script.

`scripts/start.sh` creates an ERC20Mock token that sets to it's `msg.sender` a fixed amount of tokens:

https://github.com/1Hive/conviction-voting-app/blob/2f88da71/contracts/test/ERC20Mock.sol#L13

And then when doing `npx run mock-data <dao> <cv-address>` we want to send that amount to the DAO vault, but it reverts with the `VAULT_TOKEN_TRANSFER_FROM_REVERT` error. I suspect it is because `msg.sender` is different in each script execution. Do you know how we can set it to Aragon default's account, @0xGabi @fabriziovigevani?

fabriziovigevani commented:
Hmm that's odd. 
When you run the `mockData` script it's using the already started devchain in the `start.sh` script no? It should be taking the `accounts[0]` by default if none specified as always.
Maybe there was some update in the new `@aragon/cli` version that i'm not aware of @0xGabi

sembrestels commented:
I refactored the template to include most of scripts logic, and refactored `mine-blocks` script too, so it is easier to mock and test how conviction grows and decreases.
